observability/monitor/service_monitor_gateway_plugin.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: gateway-plugin-metrics-monitor
```

Comment:
seems you have one copy in `config/prometheus` folder, here is a copy? Seems you want to have some manifest under monitor folder for user to choose from? this is ok

Reply:
changes moved to https://github.com/vllm-project/aibrix/pull/1211
will move your comment there and discuss / address